TestCases/compliance-level-3/0081-feel-getentries-function/0081-feel-getentries-function.dmn

```diff
@@ -0,0 +1,100 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<definitions namespace="http://www.montera.com.au/spec/DMN/0081-feel-getentries-function" name="0081-feel-getentries-function" id="_i9fboPUUEeesLuP4RHs4vA" xmlns="http://www.omg.org/spec/DMN/20180521/MODEL/" xmlns:di="http://www.omg.org/spec/DMN/20180521/DI/" xmlns:dmndi="http://www.omg.org/spec/DMN/20180521/DMNDI/" xmlns:dc="http://www.omg.org/spec/DMN/20180521/DC/" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+    <description>FEEL built-in function 'get entries(m)' in unspecified category</description>
+
+    <decision name="decision_001" id="_decision_001">
+        <description>Tests FEEL expression: 'get entries()' and expects result: 'null'</description>
+        <question>Result of FEEL expression 'get entries()'?</question>
+        <allowedAnswers>null</allowedAnswers>
+        <variable typeRef="list" name="decision_001"/>
+        <literalExpression>
+            <text>get entries()</text>
+        </literalExpression>
+    </decision>
+
+    <decision name="decision_002" id="_decision_002">
+        <description>Tests FEEL expression: 'get entries({a: "foo"}, {b: "bar"})' and expects result: 'null'</description>
+        <question>Result of FEEL expression 'get entries({a: "foo"}, {b: "bar"})'?</question>
+        <allowedAnswers>null</allowedAnswers>
+        <variable typeRef="list" name="decision_002"/>
+        <literalExpression>
+            <text>get entries({a: "foo"}, {b: "bar"})</text>
+        </literalExpression>
+    </decision>
+
+    <decision name="decision_003" id="_decision_003">
+        <description>Tests FEEL expression: 'get entries(null)' and expects result: 'null'</description>
+        <question>Result of FEEL expression 'get entries(null)'?</question>
+        <allowedAnswers>null</allowedAnswers>
+        <variable typeRef="list" name="decision_003"/>
+        <literalExpression>
+            <text>get entries(null)</text>
+        </literalExpression>
+    </decision>
+
+    <decision name="decision_004" id="_decision_004">
+        <description>Tests FEEL expression: 'get entries({a: "foo", b: "bar"})' and expects result: '[{a: "foo}, {b: "bar"}]'</description>
+        <question>Result of FEEL expression 'get entries({a: "foo", b: "bar"})'?</question>
+        <allowedAnswers>[{a: "foo}, {b: "bar"}]</allowedAnswers>
```

Comment:
This comment is unconsistent with the expected result.
Suggested change
From `<allowedAnswers>[{a: "foo}, {b: "bar"}]</allowedAnswers>`
To `<allowedAnswers>[{value=foo, key=a}, {value=bar, key=b}]</allowedAnswers>`